app/src/main/java/org/systers/mentorship/view/activities/SettingsActivity.kt

```diff
@@ -20,9 +26,27 @@ class SettingsActivity : BaseActivity() {
         supportActionBar?.setDisplayShowHomeEnabled(true);
 
         tvLogout.setOnClickListener {
-            preferenceManager.clear()
-            startActivity(Intent(this, LoginActivity::class.java))
-            finishAffinity()
+
+            val builder = AlertDialog.Builder(this)
+            builder.setTitle("CONFIRM LOGOUT")
+            builder.setMessage("Are you sure you want to logout ?")
+
+            builder.setPositiveButton("YES"){dialog, which ->
+
```

Comment:
Here also, remove this extra line.

Reply:
The 'Yes' text should also be in strings.xml. Also, why is everything capitalized?